src/docs/pages/introduction/getting-started/react.md

```diff
@@ -109,6 +109,107 @@ There are a few cases where native events of Web Components do not behave as exp
 </ld-notice>
 
+## Cookbook
+
+### React Router
+
+[React Router](https://reactrouter.com/) is a widely used library for routing in client-side react applications. This recipe shows you how to integrate Liquid Oxygen navigation components with React Router. Although the recipe is based on React Router, interacting with other libraries should look quite similar.
+
+Liquid Oxygen provides some sophisticated components for navigation. This recipe uses the following:
+
+- [LdSidenav](/components/ld-sidenav/) is the recommended navigation solution for most applications using Liquid Oxygen. It is flexible, responsive and provides multiple levels of navigation.
+- [LdBreadcrumbs](/components/ld-breadcrumbs/) component is a simple way to display the current location in your application.
+
+As this recipe focuses on how to put together Liquid Oxygen and React Router, we'll only highlight the relevant parts of the code. We assume you already have a basic React Router setup in place. If not, please refer to the [React Router documentation](https://reactrouter.com/en/start/tutorial).
+
+Let's get started with routing. We cannot use the `<Link>` component provided by React Router within LdSidenavNavitem, therefore we use the [useNavigate](https://reactrouter.com/en/main/hooks/use-navigate) hook to navigate to a different route.
+
+```tsx
+// Sidebar.tsx
+//...
+const navigate = useNavigate()
+
+const handleLink = useCallback(
+  (e: React.MouseEvent) => {
+    e.preventDefault()
+    const href = (e.target as HTMLLdSidenavNavitemElement).href
+    if (href) {
+      navigate(href)
+    }
+  },
+  [navigate]
+)
+
+return (
+  // ...
+  <LdSidenavNavitem
+    href={`/mathematical-foundations/discrete-mathematics`}
+    onClick={handleLink}
+  >
+    Discrete Mathematics
+  </LdSidenavNavitem>
+  // ...
+)
+// ...
+```
+
+The `href` prop of `LdSidenavNavitem` tells the component to add `LdLink` which renders a proper anchor tag. This is important for accessibility. As this would already work to navigate but bypass the client-side navigation of React Router, we also add a click handler. The click handler uses the `navigate` function from the `useNavigate` hook to navigate to the specified href. `preventDefault()` is called to prevent the default behavior of the anchor tag.
+
+`LdSidenav` provides visual indicators for an active item. We can use the `pathname` property of the `useLocation()` hook provided by React Router to determine the active route and set `selected` accordingly.
+
+```tsx
+// Sidebar.tsx
+//...
+const { pathname } = useLocation()
+
+return (
+  // ...
+  <LdSidenavNavitem
+    href={`/mathematical-foundations/discrete-mathematics`}
+    onClick={handleLink}
+    selected={pathname === '/mathematical-foundations/discrete-mathematics'} // Depending on your routes, you'll need a more sophisicated evaluation here
```

Comment:
Same as above, I'd rather use a 'real world example' than hard-coded urls.

Reply:
Same as above I try to avoid adding a layer of complexity which is not explaining routing per se. 
On the other hand: Can we expect a react developer to be able to use map on a list to generate `LdSidenavNavitem`?

As this example is about highlighting the selected item, I actually hope that the user realizes the similarity of href and the selected condition to apply it to his own problem. 🤔 

Still I think it's preferable to focus on the topic we want to explain rather then showcasing a "pseudo" real-life example which might be harder to understand.